packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1123,13 +1135,12 @@ class RotatedBox extends SingleChildRenderObjectWidget {
 ///
 /// In fact, the majority of widgets in Flutter are simply combinations of other
 /// simpler widgets. Composition, rather than inheritance, is the primary
-/// mechansim for building up widgets.
+/// mechanism for building up widgets.
 ///
 /// See also:
 ///
 ///  * [EdgeInsets], the class that is used to describe the padding dimensions.
-///  * [Center], which positions the child at its natural dimensions, centered
-///    in the parent.
+///  * The [catalog of layout containers](https://flutter.io/widgets/layout/).
```

Comment:
here and lots of other places:
I think I'd call this the catalog of layout widgets, rather than layout containers. I'm not sure what "layout container" really means.

Reply:
Done.  I called them containers because they layout a child or children.  To me, that's a container for children (like a playpen. :-)).  Widget works too, although in Flutter that's not very specific, given how many things are Widgets.  I also see that 'container' might get confused with a `Container`, however.